### PR TITLE
tests/lwip_sock_*: fix compilation error

### LINKED ISSUE #9795: tests/lwip_sock_*: compilation errors
Compilation of these tests throw the following error message on the ESP8266 platform:
```
.../tests/lwip_sock_ip/stack.c:355:36: error: 'msg.content.value' is used uninitialized in this function [-Werror=uninitialized]
     while (data_len != (msg.content.value - sizeof(ipv6_hdr_t)))
```
which seems to be indeed the case if stack variables are not initialized by default. If I understand the semantic of the source correctly, the following construct could solve the problem:
```
do {
    msg_receive(&msg);
} while (data_len != (msg.content.value - sizeof(ipv6_hdr_t)));

### PR DESCRIPTION
**Pull request by gschorcht**
Fixes #9795

### CODE REVIEW

File: tests/lwip_sock_ip/stack.c [hunk 1]
```diff
@@ -349,7 +349,7 @@ bool _check_6packet(const ipv6_addr_t *src, const ipv6_addr_t *dst,
                     uint8_t proto, void *data, size_t data_len, uint16_t netif)
 {
 #if LWIP_IPV6
-    msg_t msg;
+    msg_t msg = { .content = { .value = 0 } };
```
[the remark this answers is not in the document]
Comment (gschorcht) [reply tests/lwip_sock_ip/stack.c]: What do you mean with I324?

**Comment by gschorcht** [comment]
@miri64 Changed it for LWIP_IPV4 too.

File: tests/lwip_sock_ip/stack.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (miri64) [reply tests/lwip_sock_ip/stack.c]: line 324 ;-) but I guess that cleared up after all ;-)

**Comment by miri64** [comment]
Thanks! Please squash the two commits together to one commit.

**Comment by gschorcht** [comment]
Done

**Comment by miri64** [comment]
See, this was quick :-)

**Comment by gschorcht** [comment]
Thanks

**Pull request merged**